brut.apktool/apktool-lib/src/main/java/brut/androlib/apk/YamlReader.java

```diff
@@ -0,0 +1,198 @@
+package brut.androlib.apk;
+
+import brut.androlib.exceptions.AndrolibException;
+
+import java.io.InputStream;
+import java.util.*;
+
+public class YamlReader {
+
+    private ArrayList<YamlLine> mLines;
+    private int mCurrent = 0;
+
+    public YamlReader(InputStream in) {
+        mLines = new ArrayList<>();
+        mLines.add(new YamlLine(null));
+        read(in);
+    }
+
+    public int getLineNo() {
```

Comment:
This one is unused as well. Was it invented to help debug an error when parse error occurred?